runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessDetailsDialog/ProcessDetailsTabs/AccessTab.tsx

```diff
@@ -0,0 +1,40 @@
+import { Box, Chip, Typography } from '@mui/material';
+
+import useTranslations from '#src-app/hooks/useTranslations';
+
+import {
+    DetailsInfoTabProps,
+    ProcessDetailsTab,
+} from '../ProcessDetailsDialog.types';
+
+export const AccessTab = ({ value, process }: DetailsInfoTabProps) => {
+    const { translate } = useTranslations();
+    return (
+        <Box hidden={value !== ProcessDetailsTab.ACCESS}>
+            <Box
+                display="flex"
+                flexWrap="wrap"
+                mt={2}
+                gap={1}
+                alignItems="center"
+            >
+                <Typography variant="h5">
+                    {translate(
+                        'Component.Tile.Process.DetailsDialog.TabContent.AccessLabel.SharedWith'
+                    )}
+                </Typography>
+                {process?.processCollection?.users?.length ? (
```

Comment:
process object should be mandatory here as well, right? + what do you think about extracting this to to variable